lib/dynamoid/fields.rb

```diff
@@ -145,14 +145,14 @@ def update_attribute(attribute, value)
     #
     # @since 0.2.0
     def set_created_at
-      self.created_at = DateTime.now if Dynamoid::Config.timestamps
+      self.created_at = DateTime.now.in_time_zone(Time.zone) if Dynamoid::Config.timestamps
```

Comment:
@pboling I don't know if this sounds like a dumb question, but is this gem supposed to depend on Rails monkey patch of `Time`?

I'm considering alternatives my own wrappers for DynamoDB but my project is not a Rails project so Time.zone doesn't exist.